composer.json

```diff
@@ -36,7 +36,9 @@
         }
     },
     "scripts": {
-        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
+        "fix-cs": "php-cs-fixer fix -v --show-progress=estimating",
```

Comment:
Afaik `estimating` is deprecated, there should be a warning coming from Fixer v2